continuous_integration/test_script.sh

```diff
@@ -13,7 +13,7 @@ python -c "import numpy; print('numpy %s' % numpy.__version__)"
 python -c "import scipy; print('scipy %s' % scipy.__version__)"
 
 if [[ "$COVERAGE" == "true" ]]; then
-    if [["$MPI == "true" ]]; then
+    if [["$MPI" == "true" ]]; then
```

Comment:
Still throwing an error, I think it needs a space after [[:
`[[ "$MPI" ...`

Reply:
Are you testing locally?

Reply:
No, but travis had errors on that bash line such that mpiexec was not executed. Now it seems it is, and creates "real" errors...